rust-runtime/aws-smithy-runtime-api/src/client/runtime_components.rs

```diff
@@ -483,7 +483,7 @@ impl RuntimeComponents {
             };
         }
 
-        tracing::trace!(runtime_components=?self, cfg=?cfg, "validating final config");
+        tracing::trace!("validating final config");
```

Comment:
I'm not sure these "validating config" messages add anything now, especially if we end up logging the final outcome of an operation presumably it would show up as an error?

Reply:
Removed in https://github.com/smithy-lang/smithy-rs/pull/3664/commits/960c855807c3ad376ad862f43730acac9fcb625d